tests/typescript/handler.ts

```diff
@@ -1,10 +1,6 @@
 import { hi } from "@my-org/some-lib/index";
 import { merhaba } from "./tsx/import";
 
-export const typescript4Inject = <A extends []>(
```

Comment:
this caused an eslint error and I did not understand what it did as the function is never called